### ISSUE TITLE
Cannot generate mnist digits with the InfoGAN.py script

### ISSUE THREAD

**Issue by TeslaH2O** [eriklindernoren/Keras-GAN#7]
I tried to run the script for InfoGAN applied to mnist. However, the generated images at the end of the training phase are not even vaguely similar to digits' shapes.
Is there someone else that tried the script? Do you have my same problem? Is there anything I should try to make it work?
I have already tried to use more training epochs and reducing the learning rate but nothing worked so far....

**Comment by eriklindernoren** [comment]
Hi,

This should be fixed in 01c19e4.

Cheers,
Erik

**Comment by gdyshi** [comment]
I checked [01c19e4](https://github.com/eriklindernoren/Keras-GAN/commit/01c19e45d3d4aacb181f5b9ef1bf01b3f34f7391) and confirmed that the changes have been merged, but the problem still exists.

infogan/infogan.py
@@ -1,4 +1,4 @@
from __future__ import print_function
from __future__ import print_function, division

from keras.datasets import mnist
from keras.layers import Input, Dense, Reshape, Flatten, Dropout, multiply, concatenate
